ivadomed/utils.py

```diff
@@ -562,6 +586,40 @@ def structureWise_uncertainty(fname_lst, fname_hard, fname_unc_vox, fname_out):
     nib.save(nib_avgUnc, fname_avgUnc)
 
 
+def multiclass_dice_score(im1, im2):
+    im1 = np.asarray(im1[0, ]).astype(np.bool)
+    im2 = np.asarray(im2[0, ]).astype(np.bool)
+
+    dice_per_class = 0
+    n_classes = im1.shape[0]
+    n = 0
+    for i in range(n_classes):
+        intersection = np.logical_and(im1[i, ], im2[i, ])
+
+        # Verify that sum is not null
+        if im1[i, ].sum() + im2[i, ].sum():
+            dice_per_class += intersection.sum() / (im1[i, ].sum() + im2[i, ].sum())
+            n += 1
+    if not n:
+        n = 1
+    return (2.0 * dice_per_class) / n
+
+
+def tumor_dice_score(im1, im2):
```

Comment:
could you call `dice_score` by sending `im1[0,]` instead of `im1`, same for `im2`?

Reply:
I removed this score, I used it only for debugging and forgot to take it off :-1: